doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -334,6 +334,9 @@ Libraries for Zigbee
 * Fixes and improvements in :ref:`Zigbee Shell  <lib_zigbee_shell>` library.
 * Added :ref:`BDB command for printing install codes <bdb_ic_list>` to the :ref:`Zigbee shell <lib_zigbee_shell>` library.
 * Improve logging in :ref:`ZBOSS OSIF <lib_zigbee_osif>` library and :ref:`Zigbee Shell <lib_zigbee_shell>` library.
+* Removed support for Zigbee Green Power Proxy Basic functionality.
```

Comment:
Incorrect!
Should be:
Removed experimental support for Green Power Combo Basic functionality.

Reply:
done!